### PR TITLE
add k6 tests and format

### LINKED ISSUE #93: Stress testing Content Publishing Service API
#### Description
Implement a suite of stress tests for our Content Publishing Service API to evaluate performance and identify potential bottlenecks under high-load scenarios. Utilize stress testing tools such as Locust.io (or other tool) to simulate real-world usage and ensure the API can handle at least 256 transactions per second on each endpoint.

#### Goals
1. **Setup Stress Test Environment**: Configure a dedicated testing environment that replicates the production setup as closely as possible to ensure accurate results.
2. **Develop Stress Tests for Each Endpoint**:
    - `/api/health`: GET request to verify the health of the service.
    - `/api/asset/upload`: PUT request to simulate uploading assets.
    - `/api/content/{userDsnpId}/broadcast`: POST request to broadcast new content.
    - `/api/content/{userDsnpId}/reply`: POST request to reply to content.
    - `/api/content/{userDsnpId}/reaction`: POST request to react to content.
    - `/api/content/{userDsnpId}`: PUT and DELETE requests to update and delete content respectively.
    - `/api/profile/{userDsnpId}`: PUT request to update user profile details.
3. **Performance Targets**:
    - Each endpoint should handle a minimum of 256 transactions per second with the respective payloads.
    - Monitor response times, error rates, and system resource utilization.
4. **Workflow Simulation**:
    - After individual tests, chain requests to simulate a complete user interaction workflow across the service.
5. **Documentation and Reporting**:
    - Document performance outcomes, potential choke points, and any failures encountered during the tests.
    - Propose optimizations and enhancements based on test results.

#### Tasks
- [ ] Set up a Locust.io (or other tool) testing environment.
- [ ] Write stress test scripts for each endpoint.
- [ ] Execute stress tests on individual endpoints.
- [ ] Combine tests to simulate sequential user actions.
- [ ] Analyze and document the results.
- [ ] Create follow-up tickets for optimization based on findings.

#### Acceptance Criteria
- The stress test suite reliably measures the performance of each endpoint.
- Each endpoint meets or exceeds the threshold of 256 transactions per second.
- A comprehensive report detailing the performance, issues, and potential improvements is produced.

#### Resources
- [Locust.io Documentation](https://docs.locust.io/)
- [Content Publishing Service API Documentation](https://amplicalabs.github.io/content-publishing-service/)  

#### Notes
This might need to also work in conjunction with the [content watcher api stress testing ](https://github.com/AmplicaLabs/content-watcher-service/issues/36)as they have some functional dependencies when you get into the Workflow Simulation part.

### PR DESCRIPTION
**Pull request by claireolmstead**
closes #93

### CODE REVIEW

File: k6-test/script_sm_files.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        http_reqs: ['rate>=25']
+    },
+    noConnectionReuse: true,
+};
+
+export default function() {
+    group("/api/asset/upload medium files", () => {
+        let url = BASE_URL + `/api/asset/upload`;
+        // Request No. 1: ApiController_assetUpload medium files
+        // Currently expected to fail
```
Comment (claireolmstead) [inline k6-test/script_sm_files.js:26]: remove failure comment

File: k6-test/script_lg_files.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        http_reqs: ['rate>=0.24']
+    },
+    noConnectionReuse: true,
+};
+
+export default function() {
+    group("/api/asset/upload medium files", () => {
+        let url = BASE_URL + `/api/asset/upload`;
+        // Request No. 1: ApiController_assetUpload medium files
+        // Currently expected to fail
```
Comment (claireolmstead) [inline k6-test/script_lg_files.js:26]: remove expect to fail and change medium to large

File: k6-test/script.js [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+    noConnectionReuse: true,
+};
+
+export default function() {
+    group("/api/content/{userDsnpId}", () => {
+        let userDsnpId = '1';
+        // Request No. 1: ApiController_update with no assets
+        {
+            let url = BASE_URL + `/api/content/${userDsnpId}`;
+            // TODO: edit the parameters of the request body.
```
Comment (claireolmstead) [inline k6-test/script.js:49]: remove comment

File: k6-test/script.js [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+        }
+    });
+
+    group("/api/content/{userDsnpId}/reply", () => {
+        let userDsnpId = '1';
+
+        // Request No. 1: ApiController_reply no assets
+        {
+            let url = BASE_URL + `/api/content/${userDsnpId}/reply`;
+            // TODO: edit the parameters of the request body.
```
Comment (claireolmstead) [inline k6-test/script.js:196]: remove

**Review by mattheworris — APPROVED** [review]

**Pull request merged**